tests/atomistics/job/test_atomistic.py

```diff
@@ -69,5 +69,26 @@ def test_get_displacements(self):
         self.assertTrue(np.allclose(disp, disp_ref))
 
 
+    def test_get_structure(self):
+        """get_structure() should return structures with the exact values from the HDF files even if the size of
+        structures changes."""
+
+        # tested here with lammps as a concrete instantiation of AtomisticGenericJob
+        # have to do extra tango because Project.unpack is weird right now
+        cwd = os.curdir
+        to_dir = os.path.join(self.project.path, "..")
+        shutil.copy("tests/static/lammps_test_files/get_structure_test.tar.gz", to_dir)
```

Comment:
You haven't imported `shutil` in this file!

Reply:
Ah, the curse of copy-paste, fixed.